vendor/parser-extended/parser.mly

```diff
@@ -435,54 +444,48 @@ type let_binding =
     lb_expression: expression;
     lb_constraint: value_constraint option;
     lb_is_pun: bool;
-    lb_attributes: attributes;
-    lb_docs: docs Lazy.t;
-    lb_text: text Lazy.t;
+    lb_attributes: ext_attrs;
     lb_loc: Location.t; }
 
 type let_bindings' =
   { lbs_bindings: let_binding list;
     lbs_rec: rec_flag;
-    lbs_extension: string Asttypes.loc option }
+    lbs_has_ext: bool }
 
-let mklb first ~loc (p, args, typ, e, is_pun) attrs =
+let mklb ?(text=[]) ~docs ~loc (p, args, typ, e, is_pun) attrs =
```

Comment:
Note for later: With some work, we could store docstrings into the new `ext_attrs` and avoid encoding them in attributes.

Reply:
Do you think that's a reasonable amount of work? The "decoding" code added in Fmt_ast would be greatly reduced if we did it in this PR.

Reply:
I think we can add later as the "decoding" code is already written.

This PR was quite stale and is very old, I do not have much time to spend on ocamlformat, and I do not want this to go stale again.

Some of that code must also have been merged in #2451